packages/abstractions/package.json

```diff
@@ -38,7 +38,7 @@
     "@opentelemetry/api": "^1.2.0",
     "tinyduration": "^3.2.2",
     "tslib": "^2.3.1",
-    "uri-template-lite": "^22.1.0",
+    "uri-template-lite": "23.4.0",
```

Comment:
is there any specific reason why you're locking on that specific exact version as opposed to charet? (current major and anything under)

Reply:
Just being a bit defensive here, would you prefer a `^23.4.0` ?

Reply:
correct, thanks